nixos/modules/tasks/filesystems/zfs.nix

```diff
@@ -62,6 +62,7 @@ let
   # latter case it makes one last attempt at importing, allowing the system to
   # (eventually) boot even with a degraded pool.
   importLib = {zpoolCmd, awkCmd, cfgZfs}: ''
+    # shellcheck disable=SC2013
```

Comment:
We should add a short comment, why this was added

Reply:
I rewrote this part such that we no longer need to ignore this warning.